appveyor/Convert-FileUrisToGitHubUris.ps1

```diff
@@ -0,0 +1,120 @@
+$location=$args[0]
+$repoProvider=$env:APPVEYOR_REPO_PROVIDER
+$projectSlug=$env:APPVEYOR_PROJECT_SLUG
+$repoName=$env:APPVEYOR_REPO_NAME
+$repoCommit=$env:APPVEYOR_REPO_COMMIT
+#$repoProvider='gitHub'
+#$projectSlug='sarif-sdk'
+##$projectSlug='sarif-sdk-931m5'
+#$repoName='Rolstenhouse/sarif-sdk'
+#$repoCommit='f82ebd892917bd6157efc04b832a57964ec1f2c6'
+
+$githubBaseUri = ""
+
+if($repoProvider -eq "gitHub")
+{
+  $githubBaseUri = "https://raw.githubusercontent.com/"
+}
+
+function Rebase-Uri($originalURI)
+{
+  $caseSensitiveUri = Get-CaseSensitivePath($originalURI.SubString($originalURI.IndexOf("c:/")))
+  return $githubBaseUri + $repoName+"/"+$repoCommit+"/"+$caseSensitiveUri.SubString($caseSensitiveUri.IndexOf($projectSlug)+$projectSlug.Length+1)
```

Comment:
>$githubBaseUri [](start = 9, length = 14)

I think we should use a UriBuilder to form the URL string. I believe it will take care of the string formatting issues we're seeing (backslashes instead of forwardslashes, etc.)